genesis/genesis.json

```diff
@@ -2,18 +2,35 @@
     ]
   ],
-  "additional_storage": [],
-  "execution_version": 4,
-  "genesis_root": "0xc346a158cce093e99ab65a95c884a26629d0e4f8d00ae20bbca4bfc4b204eec2"
+  "additional_storage": {
+    "0x000000000000000000000000000000000000800f": {
+      "0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc": "0x000000000000000000000000504c4af171d1b5f31c8b8f181c21484b75110f87",
+      "0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103": "0x000000000000000000000000000000000000000000000000000000000001000c"
+    },
+    "0x000000000000000000000000000000000001000c": {
+      "0x0000000000000000000000000000000000000000000000000000000000000000": "0x000000000000000000000000000000000000000000000000000000000000800f"
+    }
+  },
+  "additional_storage_raw": [],
+  "execution_version": 3,
```

Comment:
Is this change intentional? It was 4.

Reply:
At the time when this PR was created it was 3, but things might've changed... The correct one will be included in the @AntonD3 's PR